server/routes/text2viz_routes.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { schema, TypeOf } from '@osd/config-schema';
+import {
+  HttpResponsePayload,
+  IOpenSearchDashboardsResponse,
+  IRouter,
+  RequestHandlerContext,
+} from '../../../../src/core/server';
+import { TEXT2VIZ_API } from '../../common/constants/llm';
+import { getOpenSearchClientTransport } from '../utils/get_opensearch_client_transport';
+import { ML_COMMONS_BASE_API } from '../utils/constants';
+import { getAgent } from './get_agent';
+
+// const createPrompt = (input: string, ppl: string, sampleData: string, dataSchema: string) => {
```

Comment:
Will this createPrompt be used in the future versions?

Reply:
No, I cleaned up unused codes